src/EnergyPlus/WeatherManager.cc

```diff
@@ -281,6 +282,11 @@ namespace WeatherManager {
 	Array1D< SpecialDayData > SpecialDays;
 	Array1D< DataPeriodData > DataPeriods;
 
+	std::shared_ptr< BaseGroundTempsModel > siteShallowGroundTempsPtr;
```

Comment:
@Myoldmopar @mbadams5 the shallow object is used, like you saw above by one of the other ground temp models for certain situations. @Myoldmopar feel free to chime in if you feel inclined on this. I will be out for a bit so I will respond when I get back.